.github/workflows/rainix-docker.yml

```
@@ -0,0 +1,51 @@
+        with:
+          determinate: true
+      - name: Install Nix 2/2
+        uses: DeterminateSystems/flakehub-cache-action@main
+
+      - name: Install deps
+        run: nix develop -c pnpm install --frozen-lockfile
+
+      - name: Build Price Pusher
+        run: nix develop -c pnpm turbo build --filter @pythnetwork/price-pusher
+
+      - uses: docker/setup-buildx-action@v2
```

actionlint (1.7.7)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)